nilearn/datasets/func.py

```diff
@@ -1276,3 +1276,189 @@ def fetch_mixed_gambles(n_subjects=1, data_dir=None, url=None, resume=True,
+        50 - Group ICA brain parcellations with dimensionality = 50
+        100 - Group ICA brain parcellations with dimensionality = 100
+        200 - Group ICA brain parcellations with dimensionality = 200
+        300 - Group ICA brain parcellations with dimensionality = 300
+
+        By default, network matrices data estimated from brain parcellations
+        of all dimensionalities are fetched as a seperate list.
+
+        If given an integer or list of specific integers, network matrices
+        related to that particular dimensionality of brain parcellations
+        are fetched. For example, if given as an integer 25 only data
+        corresponding to dimensionality=25 will be fetched.
+
+    timeseries: a string or list ['mutiple_spatial_regression', 'eigen_regression'] \
+        default is 'eigen_regression', optional
+        'multiple_spatial_regression' - denotes the averaged timeseries signals
+        which were extracted using mutiple spatial regression, in which full set
+        of ICA maps were used as spatial regressors against the subjects datasets
+        resulting a subject specific timeseries signals. [3]
+        'eigen_regression' - denotes the principal eigen timeseries signals which were
+        extracted using multiple spatial regression, but the subject
+        specific timeseries signals are extracted using SVD. The first
+        eigen timeseries of each subject rather than simple averaging. [4] [5]
+
+    resume: boolean, default is True
+        This parameter is required if a partly downloaded file is needed to be
+        resumed to download again.
+
+    verbose: int, default is 1
+        This parameter is used to set the verbosity level to print the message
+        to give information about the processing.
+        0 indicates no information will be given.
+
+    Returns
+    -------
+    data: sklearn.datasets.base.Bunch
+        Dictionary-like object, attributes are:
+        'Fullcorrelation': Full correlation matrices (Znet1)
+        'Partialcorrelation': Partial correlation matrices (Znet2)
+
+    Note: In output namings, 'eigen_regression' can be seen fetched as 'ts3' and
+          'multiple_spatial_regression' fetched as 'ts2'. To keep them with standard
+          Megatrawls notations.
+
+    References
+    ----------
+    For more details:
+    [1] Stephen Smith et al, HCP beta-release of the Functional Connectivity MegaTrawl.
+    April 2015 "HCP500-MegaTrawl" release.
+    https://db.humanconnectome.org/megatrawl/
+
+    Technical details:
+    [2] Smith, S.M. et al. Nat. Neurosci. 18, 1565-1567 (2015).
+
+    [3] N.Filippini, et al. Distinct patterns of brain activity in young carriers
+    of the APOE-e4 allele.
+    Proc Natl Acad Sci USA (PNAS), 106:7209-7214, 2009.
+
+    [4] S.Smith, et al. Methods for network modelling from high quality rfMRI data.
+    Meeting of the Organization for Human Brain Mapping. 2014
```

Comment:
If possible, a better reference would be welcome. I believe that this what @alexsavio pointed to in #841.